### ISSUE TITLE
code problem

### ISSUE THREAD

**Issue by starryFelix** [uci-cbcl/RP-Net#3]
corr = corr.view(batch, ht, wd, 1, ht, wd)
corr = corr  / torch.sqrt(torch.tensor(dim).float())
corr = corr.view(-1, 1, ht, wd)

Thank you very much for your work , but I did not understand this part of the code.Can you tell me ? Thank you.
